crt_portal/cts_forms/templates/forms/snippets/pagination.html

```diff
@@ -0,0 +1,38 @@
+<nav class="usa-pagination" role="navigation" aria-label="Pagination">
```

Comment:
This is great, seeing all these great a11y reccommendations from a11ymatters in action! 🙌 

https://www.a11ymatters.com/pattern/pagination/